astrapy/ops.py

```diff
@@ -13,19 +13,20 @@
 # limitations under the License.
 
 from astrapy.utils import make_request, http_methods
-from astrapy.defaults import DEFAULT_DEV_OPS_PATH_PREFIX, DEFAULT_DEV_OPS_URL
+from astrapy.defaults import DEFAULT_DEV_OPS_API_VERSION, DEFAULT_DEV_OPS_URL
 
 import logging
 
 logger = logging.getLogger(__name__)
 
 
 class AstraDBOps:
-    def __init__(self, token, dev_ops_url=None):
+    def __init__(self, token, dev_ops_url=None, dev_ops_api_version=None):
```

Comment:
I would always, here as in db.py, first strip slashes from both ends of such a string, then manually put them between when joining. Probably three people will start passing `/v1`, `v1` and `/v1/` here and we want them all to be happy